examples/multi_physics/piezo_elasticity_macro.py

```diff
@@ -0,0 +1,165 @@
+r"""
+Piezo-elasticity problem - homogenization of a piezoelectric linear elastic
+matrix with embedded metalic electrodes, see [1] for details.
+
```

Comment:
Could you explain a bit the usage (what is the grid argument to define() and that it can be set with -D option to simple.py)?

Reply:
I have removed grid0 argument, it was important in a more complex example, not in this one.
Argument eps0 is explained a little bit.